frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { ListGroup } from 'react-bootstrap';
+import { IParcel } from 'actions/parcelsActions';
+import { Label } from 'components/common/Label';
+import './InfoSlideOut.scss';
+
+export type IParcelAttributes = {
+  /** the selected parcel information */
+  parcelInfo: IParcel | null;
+};
+
+/**
+ * Displays parcel specific information needed on the information slide out
+ * @param parcelInfo the selected parcel data
+ */
+export const ParcelAttributes: React.FC<IParcelAttributes> = ({ parcelInfo }) => {
+  let formatAssessed;
+  if (parcelInfo?.assessed !== '') {
+    formatAssessed = '$' + new Intl.NumberFormat('en-US').format(parcelInfo?.assessed as number);
```

Comment:
format the assessed value into $___,___ format (no cents)

Reply:
Id like to see you either use one of the util functions for currency formatting or add one if you have a new use case.

Reply:
I didn't realize we had some, I'll do that!